docs/README.rst

```diff
@@ -5,34 +5,36 @@ suricata-formula
 
 |img_travis| |img_sr|
 
-.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/suricata-formula.svg?branch=master
+.. |img_travis| image:: https://travis-ci.com/alias454/suricata-formula.svg?branch=master
    :alt: Travis CI Build Status
    :scale: 100%
-   :target: https://travis-ci.com/saltstack-formulas/suricata-formula
+   :target: https://travis-ci.com/alias454/suricata-formula
```

Comment:
These badges will actually work from your repo (currently showing `build unknown`).  These can always be fixed when transferring/forking to the org.